sourcegraph/sourcegraph
# Enable/rollout "Getting started tour for visitors" on dotcom mode

Closes https://github.com/sourcegraph/sourcegraph/issues/27177
Closes https://github.com/sourcegraph/sourcegraph/issues/30258

> Note: also cleans up concluded "search notebook examples" experiment https://github.com/sourcegraph/sourcegraph/issues/23096

## Description
This PR:
- Enables "Getting started tour for visitors" (previously was in [AB test](https://github.com/sourcegraph/sourcegraph/issues/30258)) on dotcom mode
- Renames "onboarding tour" prefix to "getting started tour" to avoid confusion with "search onboarding tour"
- Removes old/unused search/notebook examples from LoggedOutHomePage

## Test plan

- `SOURCEGRAPH_API_URL=https://sourcegraph.com sg start web-standalone`
- Open home page as not signed in user
- Check that "Getting Started" section is there and other "search onboarding tour" is not shown up
- Sign in and check that "getting started" section doesn't appear



## Follow-up
Delete "getting-started-tour" [feature flag](https://docs.sourcegraph.com/dev/how-to/use_feature_flags#disable-or-delete-the-feature-flag)
